challenge_serving_drinks/src/challenge_serving_drinks/get_order.py

```diff
@@ -0,0 +1,147 @@
+# ROS
+import smach
+
+# TU/e Robotics
+import robot_smach_states as states
+import robot_smach_states.util.designators as ds
+
+from robocup_knowledge import knowledge_loader
+from robot_skills.robot import Robot
+
+# Serving drinks
+from .sd_states import AskDrink, DetectWaving
+
+# Knowledge
+COMMON_KNOWLEDGE = knowledge_loader.load_knowledge("common")
```

Comment:
Don't use capital letters, it is used for global constants.

Reply:
Will be reflected in the new PR (styling)